PyInstaller/hooks/hook-gi.repository.GdkPixbuf.py

```diff
@@ -40,8 +40,7 @@
         prefix = '"' + libdir
         plen = len(prefix)
         
-        # TODO: python3 probably will break here
-        for line in cachedata.splitlines():
+        for line in str(cachedata).splitlines():
```

Comment:
I assume, this can break as soon as there is a non-ascii character in `cachedata`. Try this:
```
>>> b1 = b'abcd\xc3\xa4lmn' # contains a-umlaut
>>> str(b1)
"b'abcd\\xc3\\xa4lmn'"
>>> print(str(b1))
b'abcd\xc3\xa4lmn'
```